homeassistant/components/litejet/trigger.py

```diff
@@ -88,10 +90,12 @@ def pressed():
                 hass, pressed_more_than_satisfied, dt_util.utcnow() + held_more_than
             )
 
-    def released():
+    def released() -> None:
         """Handle the release of the LiteJet switch's button."""
         nonlocal cancel_pressed_more_than, pressed_time
         nonlocal held_less_than, held_more_than
+        if pressed_time is None:
+            return
```

Comment:
Is this a real scenario?
Can `released` be called even if `pressed` hasn't been called? Or is it only to satisfy `mypy`?

If it is a real scenario, then maybe this should be considered a bugfix and be move to a preliminary PR.

Reply:
It is a very unrealistic edge case. It would only happen if someone was holding down a wall switch while HA started up. I only added it to satisfy mypy.